airbyte-cdk/python/pyproject.toml

```
@@ -1,8 +1,91 @@
-# Defines Python build system settings.
-[build-system]
-requires = [
-    "setuptools>=42",
-    "wheel"
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+[tool.poetry]
+name = "airbyte_cdk"
+description = "A framework for writing Airbyte Connectors."
+authors = ["Airbyte <contact@airbyte.io>"]
+license = "MIT"
+readme = "README.md"
+keywords = ["airbyte", "connector-development-kit", "cdk"]
+documentation = "https://docs.airbyte.io/"
+repository = "https://github.com/airbytehq/airbyte"
+
+# The version of the airbyte-cdk package is used at runtime to validate manifests. That validation must be
+# updated if our semver format changes such as using release candidate versions.
+version = "0.54.0"
+
+exclude = ["unit_tests"]
+include = [
+    "airbyte_cdk",
+    "airbyte_cdk/py.typed",
+    "airbyte_cdk/sources/declarative/declarative_component_schema.yaml"
+]
+
+[tool.poetry.urls]
+"Tracker" = "https://github.com/airbytehq/airbyte/issues"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
```

Should be able to use `^` here instead

```suggestion
python = "^3.8.1
```